cmake/sysbuild/suit.cmake

```diff
@@ -308,7 +308,7 @@ function(suit_create_package)
     message(STATUS "Found ${image} manifest template: ${INPUT_ENVELOPE_JINJA_FILE}")
 
     suit_set_absolute_or_relative_path(${INPUT_ENVELOPE_JINJA_FILE} ${app_config_dir} INPUT_ENVELOPE_JINJA_FILE)
-    sysbuild_get(target IMAGE ${image} VAR CONFIG_SUIT_ENVELOPE_TARGET KCONFIG)
+    sysbuild_get(TARGET IMAGE ${image} VAR CONFIG_SUIT_ENVELOPE_TARGET KCONFIG)
```

Comment:
If I correctly remember - all local CMake variables should be written in lowercase...